internal/docs/design.md

```diff
@@ -141,9 +141,9 @@ concrete type backed by GCS.
 -   Order arguments that are less likely to change across multiple calls to the
     constructor before ones that are likely to change. For example, connection
     and authorization related arguments should go before names, so
-    ```OpenBucket(ctx, client, "mybucket")``` instead of ```OpenBucket(ctx,
-    "mybucket", client)```.
--   All public constructors should take an `Options` struct (see next section).
+    `OpenBucket(ctx, client, "mybucket")` instead of `OpenBucket(ctx,
+    "mybucket", client)`. - All public constructors should take an `Options`
```

Comment:
It looks like `mdformat` pulled this bullet point up incorrectly.

Reply:
Done.